doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,164 +3,239 @@
 
 This section lists known issues with {PRODUCT} {PRODUCT_VERSION}.
 
-ifdef::DM[]
+COMMENT: Before finalizing, RHPAM or RHDM jiras must be created for each issue. Original jiras are linked in this draft so SMEs can find their issues easily and will be removed before publishing.
 
-== Upgrading
 
-.After upgrading to {PRODUCT} 7.3.0, rules do not fire.
+ifdef::DM[]
 
-Issue: If you upgrade to {PRODUCT} 7.3.0, rules do not fire because of a change to packages in the `kmodule.xml` file. For more information, see
-https://access.redhat.com/solutions/4014541[Rules don't fire after upgrading to RHDM 7.3.0].
 endif::[]
 
 == {CENTRAL}
-ifdef::DM[]
-.All line separators in the right-hand side of rules in the spreadsheet are escaped [https://issues.jboss.org/browse/RHDM-903[RHDM-903]]
 
-Issue: If you have multiple lines in the *ACTION* column of a spreadsheet, all line separators are converted to `\n`. This conversion causes a build error.
+.You cannot add a new group in {CENTRAL} integrated with RH-SSO [https://issues.jboss.org/browse/RHPAM-2036[RHPAM-2036]]
+
+Issue: The following exception occurs when you try to add a new group in {CENTRAL} if {CENTRAL} is integrated with RH-SSO 7.3.0:
+[source]
+----
+java.lang.NoSuchMethodError:org.keycloak.representations.idm.RoleRepresentation.setScopeParamRequired(Ljava/lang/Boolean;)
+----
```

Comment:
I would add that as a workaround users should add the group in RH-SSO directly rather than using business-central user and group management.

Reply:
Added to the draft.

Reply:
Done.